reference/public-ip-addresses.md

```diff
@@ -18,3 +20,23 @@ North American region (us.platform.sh):
 These addresses are stable, but not guaranteed to never change. Prior to
 any future change, all affected customers will receive ample warning.
 
+## Inbound IP addresses
+
+Platform.sh regions accept connections on a limited number of IP
+addresses. You may need to whitelist them on your corporate firewall
+to enable SSH connections.
```

Comment:
If you were behind a firewall, you would need to add platformsh's IPs in the outgoing rules.